src/js/standard/icons/volume.js

```diff
@@ -0,0 +1,35 @@
+import globalStyles from '../global-styles.scss';
+import createClassNameString from '../../lib/create-classname-string';
+
+export const volumeMuteIcon = (cssClass) =>`
```

Comment:
Again, I personally prefer named functions for exports, but maybe that's just me.